src/components/App.js

```diff
@@ -1,14 +1,105 @@
-import React from 'react';
+/* eslint-disable no-unused-vars */
+import React, { useState, Component, Fragment } from 'react';
+import ButtonPannel from './ButtonPanel';
 import Display from './Display';
-import ButtonPanel from './ButtonPanel';
-
-const App = function app() {
-  return (
-    <>
-      <Display />
-      <ButtonPanel />
-    </>
-  );
-};
+import Button from './Button';
+import calculate from '../logic/calculate';
+
+const buttons = [
+  'AC',
+  '+/-',
+  '%',
+  '/',
+  '9',
+  '8',
+  '7',
+  '*',
+  '6',
+  '5',
+  '4',
+  '+',
+  '3',
+  '2',
+  '1',
+  '-',
+  '0',
+  '.',
+  '=',
+];
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      total: null,
+      prev: '0',
+      next: '0',
+      operator: null,
+    };
+  }
+
+  handledisplay = (prev, next, tot, op) => {
+    let ans;
+    if (tot) {
+      ans = tot;
+    } else if (!op) {
+      ans = prev;
+    } else {
+      ans = next;
+    }
+    return ans;
+  };
+
+  handleClick = e => {
+    const {
+      total, next, operator, prev,
+    } = this.state;
+    const res = calculate(
+      {
+        total,
+        next,
+        operator,
+        prev,
+      },
+      e.target.value,
+    );
+
+    this.setState({
+      ...res,
+    });
+  };
+
+  render() {
+    const {
+      prev, next, total, operator,
+    } = this.state;
+    return (
+      <>
+        <div className="App">
+          <div className="Appcalculator">
+            <Display
+              current={this.handledisplay(prev, next, total, operator)}
+            />
+            <ButtonPannel>
+              {buttons.map(button => {
+                const operators = ['+', '-', '*', '/', '%', 'X²'];
+                const isOperator = operators.includes(button);
+                const isAcBtn = button === 'AC';
+                return (
+                  <Button
+                    onClick={this.handleClick}
+                    key={button}
+                    name={button}
+                    operator={isOperator}
+                    acBtn={isAcBtn}
+                  />
+                );
+              })}
+            </ButtonPannel>
+          </div>
+        </div>
+      </>
+    );
+  }
+}
 
```

Comment:
- [ ] The `handleClick` function should be passed down as a `clickHandler ` ( or `onClick` in your own case) property to `ButtonPanel`. In your application, you have implemented the `Button` component in the `App` component and passed the `handleClick` property to the `Button` component. Please let's correct it. 
NB: By the time you make the correction, the` buttons` variable you have in the `App` component will no longer be required. 

- [ ] Please, do not disable eslint. Try to resolve the issue by finding solutions. In this case, the eslint error is `unused vars` which simply means that you have declared a variable you did not use. Please let's fix it.